views/base.html.twig

```diff
@@ -98,6 +98,13 @@
         </div>
     {% endif %}
     <div class="{% block bodycontainer %}container{% endblock %}">
+        {% if global_notices is defined and global_notices|length > 0 %}
+            {% for notice in global_notices %}
+                <div class="alert alert-warning">
```

Comment:
I debated when implementing the one for the dashboard about whether we needed any other variant of this, e.g. `alert-info`. I'm guessing we would only want to use this for urgent information?

Reply:
I thought about options for how the notice is displayed (different alert classes or different presentations altogether) but decided that for now warning makes sense.  If this changes in the future, we can add some more options, but it doesn't seem necessary now.